### DISCUSSION TITLE
2023-09-08

### DISCUSSION THREAD

**Discussion by MoonJeWoong** [woowacourse-teams/2023-haru-study#482] [daily_meeting]
### 오늘의 할 일
- 사용자 유치 계획 discussion에 하나씩 아이디어 남기기 (9/8)
  - https://github.com/woowacourse/retrospective/discussions/40
- 분야 별 SSE 기능 관련 논의 정리
- 팀 회의
  - 개인용 서비스를 유지하는 방향에 대해 논의
  - SSE 관련해서 팀 전체 논의 & 결정

### 개인 일정
- 노아
  - 4시 근로

**Comment by jaehee329** [comment]
### 체크인
6점 - 내일이 주말이네요
### 체크아웃
3점 - 코찔찔맨

**Comment by aak2075** [comment]
### 체크인

10점 - 불금이네요

**Reply by nlom0218** [reply]
오늘 한 잔 하나?

**Comment by nlom0218** [comment]
### 체크인

7점 - 테코톡이 다가오네요...

### 체크아웃

6점 - 리프레시가 필요해요!

**Comment by woo-jk** [comment]
### 체크인
8점 - 날이 좋아서

### 체크아웃
9점 - 모든 날이 좋아서

**Comment by woosung1223** [comment]
### 체크인
6점 - 금요일이지만 너무 졸리군여

### 체크아웃
6점 - 이상한 Filter 버그 때문에 6점 드리겠습니다..

**Comment by MoonJeWoong** [comment]
### 체크인
8점 - 오늘 할 일이 많은데 강의까지 있네요. 정신 똑디 차리고 처리해야겠습니다.

### 체크아웃
8점 - SSE 이슈가 터져서 점수 떨어질 뻔 했는데 그래도 원인이 어딘지는 알아서 그대로 유지했습니당

**Comment by yeopto** [comment]
### 체크인

10점 - 숙취가 많습니다.

### 체크아웃

10점 - 구현이 쉽지않지만 이것또한 이겨낼 생각에 기쁩니다.
